app/main.js

```diff
@@ -202,24 +202,14 @@ function getTrayMenu () {
     }
   }, {
     type: 'separator'
-  }, {
-    label: 'Settings',
+  })
+
+  trayMenu.push({
+    label: 'Start a break!',
     click: function () {
-      showSettingsWindow()
+      planMicrobreak(200)
```

Comment:
Yeah, that's what I was thinking: whether to fast forward or to have extra break. 

Because `planMicrobreak` uses `startMicrobreakTimer` which is used globally for `main.js`, so your implementation behaves like fast-forward scenario. 

Also we need to consider that break could be already started when someone clicks the menu item.